tutorials/advanced_features/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -1,36 +1,49 @@
 {
```

Comment:
<p>"&nbsp;(<code>True</code>&nbsp;by default for&nbsp;<code>NoisyResults</code>)<strong>."</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>I remove it?</p>

Reply:
No, it's just the `.` that is in the wrong place